pkg/sql/pgwire/conn.go

```diff
@@ -1361,69 +1365,59 @@ func (c *conn) CreateStatementResult(
 	formatCodes []pgwirebase.FormatCode,
 	conv sessiondata.DataConversionConfig,
 ) sql.CommandResult {
-	res := c.makeCommandResult(descOpt, pos, stmt, formatCodes, conv)
```

Comment:
Is makeCommandResult used anywhere else now?

Reply:
No, it was only called from one place before too. I'm ok not manually inlining it though, as I think it's definition should live next to `conn.newMiscResult`.

Reply:
Can we just remove it?

Reply:
Well my point before was that it should live next to `conn.allocCommandResult` and `conn.newMiscResult`, but I guess we can move them all. Let me know if you like the second commit I just added. I'll squash it if you do.

Reply:
I'm not sure we're talking about the same thing, and there's a good chance I'm confused. I mean can we remove makeCommandResult if it's not used anywhere else? I have no opinion about the inlining manually. I think your first commit is perfectly fine.

Reply:
Ugh yeah I just misread things. It's already been removed and that didn't register. First commit is fine. Move along.

Reply:
👍